backend/dal/Services/ResearchFileService.cs

```diff
@@ -0,0 +1,62 @@
+using System.Collections.Generic;
+using System.Security.Claims;
+using Microsoft.Extensions.Logging;
+using Pims.Dal.Entities;
+using Pims.Dal.Helpers.Extensions;
+using Pims.Dal.Repositories;
+using Pims.Dal.Security;
+
+namespace Pims.Dal.Services
+{
+    public class ResearchFileService : IResearchFileService
+    {
+        private readonly ClaimsPrincipal _user;
+        private readonly ILogger _logger;
+        private readonly IResearchFileRepository _researchFileRepository;
+        private readonly IPropertyRepository _propertyRepository;
+
+        public ResearchFileService(ClaimsPrincipal user, ILogger<ResearchFileService> logger, IResearchFileRepository researchFileRepository, IPropertyRepository propertyRepository)
+        {
+            _user = user;
+            _logger = logger;
+            _researchFileRepository = researchFileRepository;
+            _propertyRepository = propertyRepository;
+        }
+
+        public PimsResearchFile Add(PimsResearchFile researchFile)
+        {
+            _logger.LogInformation("Adding research file...");
```

Comment:
we should discuss our use of logging, we aren't doing this enough and it would be nice to come up with some patterns.